test-case/check-playback.sh

```diff
@@ -65,6 +68,9 @@ func_lib_setup_kernel_last_line
 func_lib_check_sudo
 func_pipeline_export $tplg "type:playback"
 
+sleep_lst=(0.5 1.9 2 2.1 2.9 3 3.1 5) #And it is 3s for SOF PCI device, 2s for HDA codec in future (not merged).
```

Comment:
What is "it" 3s and what is 2s? The timeout before suspend? 

By moving the comment to the previous line they can become shorter than 80 characters. `shellcheck` should show this, please try it.